petisco/controller/controller_handler.py

```diff
@@ -210,6 +217,11 @@ def run_controller(*args, **kwargs) -> Tuple[Result, float]:
 
                 self.event_bus.publish(request_responded)
 
+            if self.inject_apm_metadata and apm_extension_is_installed():
+                import elasticapm
+
+                elasticapm.set_custom_context({"info_id": info_id.to_dict()})
```

Comment:
It would be nice if we can keep the same event structure than in other events. The info_id is under the fields ["data"]["meta"]["info_id"]. Is it possible to change it?

Reply:
As we have seen in the documentation, this does not seem possible 😔